src/test/java/org/fundation/coding/adrian/katas/CesarCypherTest.java

```diff
@@ -0,0 +1,32 @@
+package org.fundation.coding.adrian.katas;
+
+import org.fundation.coding.adrian.katas.cyphers.CesarCrypto;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Cesar cypher.
+ */
+public class CesarCypherTest {
+    /**
+     * test1.
+     */
+    @Test
+    public void test1() {
+        CesarCrypto cesarCrypto = new CesarCrypto();

```

Comment:
You are repeating this line of code in all your tests
Please move it to a @Before method.